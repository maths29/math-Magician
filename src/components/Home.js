import '../styles/home.css';
import React from 'react';
import Header from './Header';

const Home = () => (
  <>
    <Header />
    <div>
      <h2>Welcome to our page</h2>
      <div className="paraDiv">
        <p className="paragraph">
          Lorem ipsum dolor, sit amet consectetur adipisicing elit.
          Voluptates consequatur nisi odio dolores inventore cum harum repudiandae
          atque asperiores deleniti adipisci soluta placeat minima cumque officiis impedit,
          quam libero laboriosam sunt. Ex labore soluta laboriosam laborum aliquam quisquam
          quidem, quod beatae atque minima, nemo eum odio velit laudantium asperiores odit
          reiciendis ipsa animi maiores! A maiores fugit voluptatum odio nostrum autem rem
          repellat? Quod unde fuga pariatur id in adipisci ex nam voluptate quis quos? Enim,
          quasi nemo! Obcaecati numquam at consectetur! Dolore ratione voluptatibus ipsam
          facilis?
        </p>
        <p className="paragraph">
          Lorem ipsum dolor, sit amet consectetur adipisicing elit.
          Voluptates consequatur nisi odio dolores inventore cum harum repudiandae
          atque asperiores deleniti adipisci soluta placeat minima cumque officiis impedit,
          quam libero laboriosam sunt. Ex labore soluta laboriosam laborum aliquam quisquam
          quidem, quod beatae atque minima, nemo eum odio velit laudantium asperiores odit
          reiciendis ipsa animi maiores! A maiores fugit voluptatum odio nostrum autem rem
          repellat? Quod unde fuga pariatur id in adipisci ex nam voluptate quis quos? Enim,
          quasi nemo! Obcaecati numquam at consectetur! Dolore ratione voluptatibus ipsam
        </p>
      </div>
    </div>
  </>
);

export default Home;
