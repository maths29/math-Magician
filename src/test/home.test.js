import React from 'react';
import { render } from '@testing-library/react';
import Home from '../components/Home';

test('Home renders correctly on the browser', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
