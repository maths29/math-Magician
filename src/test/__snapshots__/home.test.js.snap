// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly on the browser 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Welcome to our page
    </h2>
    <div
      class="paraDiv"
    >
      <p
        class="paragraph"
      >
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptates consequatur nisi odio dolores inventore cum harum repudiandae atque asperiores deleniti adipisci soluta placeat minima cumque officiis impedit, quam libero laboriosam sunt. Ex labore soluta laboriosam laborum aliquam quisquam quidem, quod beatae atque minima, nemo eum odio velit laudantium asperiores odit reiciendis ipsa animi maiores! A maiores fugit voluptatum odio nostrum autem rem repellat? Quod unde fuga pariatur id in adipisci ex nam voluptate quis quos? Enim, quasi nemo! Obcaecati numquam at consectetur! Dolore ratione voluptatibus ipsam facilis?
      </p>
      <p
        class="paragraph"
      >
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptates consequatur nisi odio dolores inventore cum harum repudiandae atque asperiores deleniti adipisci soluta placeat minima cumque officiis impedit, quam libero laboriosam sunt. Ex labore soluta laboriosam laborum aliquam quisquam quidem, quod beatae atque minima, nemo eum odio velit laudantium asperiores odit reiciendis ipsa animi maiores! A maiores fugit voluptatum odio nostrum autem rem repellat? Quod unde fuga pariatur id in adipisci ex nam voluptate quis quos? Enim, quasi nemo! Obcaecati numquam at consectetur! Dolore ratione voluptatibus ipsam
      </p>
    </div>
  </div>
</DocumentFragment>
`;
